test/controllers/api/v2/activation_keys_controller_test.rb

```diff
@@ -295,6 +295,16 @@ def test_should_not_update_existing_with_invalid_max_hosts
       assert_match 'Validation failed: Max hosts is not a number', @response.body
     end
 
+    def test_should_not_update_with_invalid_release
+      activation_key = ActivationKey.new(
+        :name => 'new key', :organization => @organization
+      )
+      assert activation_key.save
```

Comment:
We're not testing the model's validity here, so I'd say the `assert .. save` isn't needed. (It's probably not needed in the previous test either.) I'd just go with `ActivationKey.create!`